### ISSUE TITLE
SyntaxError: Unexpected token < in JSON at position 0

### ISSUE THREAD

**Issue by k1enttt** [k1enttt/web-smart-water#14]
# Summary

This error does not show when i ran `npm run build` locally. It shows in build logs of vercel after i push this commit 48f22cd46ff1a4bf87b741b276c0082358037d1f.

# Standard debugging steps

## How to reproduce?

- Checkout to `mqtt` branch and pull commit 48f22cd46ff1a4bf87b741b276c0082358037d1f
- Deploy this commit to vercel
- The error will be shown at build log of vercel

## Excepted behaviour

- Vercel run `npm run build` and get the same logs when i run locally

## Stacktrace

```shell
SyntaxError: Unexpected token < in JSON at position 0
    at JSON.parse (<anonymous>)
    at parseJSONFromBytes (node:internal/deps/undici/undici:5584:19)
    at successSteps (node:internal/deps/undici/undici:5555:27)
    at fullyReadBody (node:internal/deps/undici/undici:1665:9)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async specConsumeBody (node:internal/deps/undici/undici:5564:7)
    at async d (/vercel/path0/.next/server/app/activity-logs/page.js:18:4568)
    at async p (/vercel/path0/.next/server/app/activity-logs/page.js:18:4783) {
  digest: '4156480153'
}
```

# Other details

**Comment by k1enttt** [comment]
This error show because the fetch url i set is "http://localhost:3000/api/..." so Vercel can not fetch and return a html of error page.

**Comment by k1enttt** [comment]
Fixed by [871b813](https://github.com/k1enttt/web-smart-water/commit/871b813d0e8dc4045008200cbcbfa77f9f88e27f)
